typing/includemod.ml

```diff
@@ -165,6 +164,12 @@ let kind_of_field_desc = function
   | Field_class _ -> "class"
   | Field_classtype _ -> "class type"
 
+(** Map indexed field names. Allow to preserve namespaces. *)
```

Comment:
This comment isn't very clear---can you reword?

Reply:
The new one is : 
```
(** Map indexed by both field types and names. 
    This avoids name clashes between different sorts of fields 
    such as values and types. *)
```